File: TransCoder_tokenized_test_set_functions/java/N_BONACCI_NUMBERS_1.java
static void bonacciseries ( int n, int m ) {
  int a [ ] = new int [ m ];
  for ( int i = 0;
  i < m;
  i ++ ) a [ i ] = 0;
  a [ n - 1 ] = 1;
  a [ n ] = 1;
  for ( int i = n + 1;
  i < m;
  i ++ ) a [ i ] = 2 * a [ i - 1 ] - a [ i - n - 1 ];
  for ( int i = 0;
  i < m;
  i ++ ) System . out . print ( a [ i ] + " " );
}

File: TransCoder_tokenized_test_set_functions/python/N_BONACCI_NUMBERS_1.py
def bonacciseries ( n, m ) :
    a = [ 0 for i in range ( m ) ]
    a [ n - 1 ] = 1
    a [ n ] = 1
    for i in range ( n + 1, m ) :
        a [ i ] = 2 * a [ i - 1 ] - a [ i - n - 1 ]
    for i in range ( 0, m ) :
        print ( a [ i ], end = " " )
    
